apps/dialtone-documentation/docs/_data/site-nav.json

```diff
@@ -562,8 +562,8 @@
           "link": "/components/emoji-text-wrapper.html"
         },
         {
-          "text": "Icon",
-          "link": "/components/icon.html"
+          "text": "Empty states",
+          "link": "/components/empty-states.html"
```

Comment:
state, rather than "states" to be consistent with all our other pages, and the actual page heading.